CRM/Civicase/Hook/Tokens/AddContactTokens.php

```diff
@@ -50,11 +52,11 @@ public function __construct(
   public function run(array &$tokens) {
     foreach ($this->contactFieldsService->get() as $field) {
       $tokens[self::TOKEN_KEY]['current_user.contact_' . $field] =
-        ts('Current User ' . ucwords(str_replace("_", " ", $field)));
+        E::ts('Current User ') . ucwords(str_replace("_", " ", $field));
```

Comment:
You can also do 
```php

 E::ts('Current User %1', array(
      1 => ucwords(str_replace("_", " ", $field),
    ));
```

Reply:
Kept it like it was before, and just applied the newly created function.